### PR TITLE
Removed tag added for testing and included cron job scheduling

### PR DESCRIPTION
**Pull request by chandrams**

### CODE REVIEW

**Comment by chandrams** [comment]
Worked with Gabor Burges to install the github app on the repo and I see the PR checks triggered as part of #112.  I have removed the temporary commitsha tag that I had added for testing and included cron job scheduling for this workflow. 

@patilsuraj767 @kgaikwad - Can you please review the changes, thanks.

File: .github/workflows/test-ros-kruize-rm.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 name: Test ROS OCP with latest Kruize remote monitoring
 
 on:
+  schedule:
+    - cron: "5 0 * * *"
```
Comment (kgaikwad) [inline .github/workflows/test-ros-kruize-rm.yml:5]: @chandrams do we really need to run cron job every 5 min?

**Review by kgaikwad — CHANGES_REQUESTED** [review]
@chandrams, added a inline comment about frequency of job run. 
As #112  is test PR, can we close it? Also, could you please confirm whether new PR will be raised by bot email `insights@redhat.com`?

**Comment by chandrams** [comment]
> @chandrams, added a inline comment about frequency of job run. As #112 is test PR, can we close it? Also, could you please confirm whether new PR will be raised by bot email `insights@redhat.com`?

@kgaikwad - I did add a comment to close #112 but couldn't find the option to close. I will update the email id, thanks for pointing it out.

File: .github/workflows/test-ros-kruize-rm.yml (hunk 1 above)
Comment (kgaikwad) [reply .github/workflows/test-ros-kruize-rm.yml:5]: @chandrams  my bad! I have considered `"5 * * * *"` & ignored zero value. +1

**Comment by kgaikwad** [comment]
\retest

**Comment by kgaikwad** [comment]
@chandrams  - also with this change, will it raise a separate PR everytime?

**Review by kgaikwad — APPROVED** [review]
LGTM :+1:

**Pull request merged**